test/lib/open_days_test.rb

```diff
@@ -0,0 +1,72 @@
+require "test_helper"
+
+class ActivityNotifierTest < ActiveSupport::TestCase
+  TIME_SLOTS = [
+    OpenStruct.new(day: "Thursday", from: 18, to: 20),
+    OpenStruct.new(day: "Saturday", from: 11, to: 13)
+  ].freeze
+
+  test "returns possible time slots" do
+    Timecop.freeze(DateTime.new(2020, 9, 21)) do
```

Comment:
The [built-in Rails travel_to helper](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel) can probably be used here instead of adding Timecop to the app.